tests/wpt/web-platform-tests/tools/wptserve/wptserve/config.py

```diff
@@ -316,10 +316,9 @@ def _get_ssl_config(self, data):
         kwargs = data["ssl"].get(ssl_type, {})
         self._ssl_env = ssl_cls(self.logger, **kwargs)
         self._ssl_env.__enter__()
-        if self._ssl_env.ssl_enabled:
```

Comment:
@jdm I think the problem is that the python script keeps thinking that I am in a `NoSSLEnvironment` even after when I add the openssl binary directory to PATH.